source/core/src/main/com/csse3200/game/files/FileLoader.java

```diff
@@ -15,6 +15,12 @@
 public class FileLoader {
   private static final Logger logger = LoggerFactory.getLogger(FileLoader.class);
   static final Json json = new Json();
+  private static final String saveRootPath = switch (System.getProperty("os.name")
```

Comment:
actually saveRootPath would probably need to be public, because now you would need to replace Gdx.files.external(___).exists() with Gdx.files.external(saveRootPath + ___).exists() and that could be done outside FileLoader

Reply:
you can just get the file handle first and get it after... this way the user of this class doesn't need to know anything they don't need to know.